qiime2/metadata/tests/test_metadata.py

```diff
@@ -1393,14 +1324,14 @@ def test_with_artifacts(self):
         md = Metadata(pd.DataFrame(
             {'col1': [1, 2, 3], 'col2': ['foo', 'bar', 'baz']},
             index=pd.Index(['a', 'b', 'c'], name='id')))
-        md._add_artifacts([artifact1, artifact2])
+        md._add_artifacts((artifact1, artifact2))
 
         obs = md.filter_ids({'a', 'c'})
 
         exp = Metadata(pd.DataFrame(
             {'col1': [1, 3], 'col2': ['foo', 'baz']},
             index=pd.Index(['a', 'c'], name='id')))
-        exp._add_artifacts([artifact1, artifact2])
+        exp._source_artifact = (artifact1, artifact2)
```

Comment:
This should probably still be `_add_artifacts` not that it matters because this line of code will never be reached because this will break on line 1327. I left this test failing like this because I'm not really sure what to do with it right now. We would need to create a `Metadata` object with both of those artifacts via a merge that didn't contain the column that was filtered out. This is completely doable, but it feels somewhat fishy to me. I'm probably just overthinking this though.

Reply:
Same as above goes for the equivalent test in `test_metadata_column.py`

Reply:
Yeah, this test (`test_with_artifacts`) is probably not relevant now, since we are removing the ability to construct metadata from multiple artifacts at once.